src/lib/PPrint.hs

```diff
@@ -294,6 +335,12 @@ instance PrettyPrec Atom where
     RecordTy items -> prettyExtLabeledItems items (line <> "&") ":"
     VariantTy items -> prettyExtLabeledItems items (line <> "|") ":"
 
+unInline :: Text -> Maybe Text
+unInline t = do
```

Comment:
Why is it called `unInline` when it removes the parens?

Reply:
Oh, that's a typo! I think I meant `unInfix`. I'll change it to that (unless you had a different suggestion?)

Reply:
Changed to `fromInfix`